src/Raven.Studio/typescript/components/pages/resources/databases/partials/NoDatabases.tsx

```diff
@@ -29,14 +20,21 @@ export function NoDatabases() {
                 {isOperatorOrAbove && (
                     <div className="d-flex gap-1">
                         <FlexGrow />
-                        <Button outline color="primary" onClick={withPreventDefault(newDatabase)}>
+                        <Button outline color="primary" onClick={() => setCreateDatabaseMode("regular")}>
                             Create new database
                         </Button>
                         <div>or</div>
-                        <Button outline color="primary" onClick={withPreventDefault(newDatabaseFromBackup)}>
+                        <Button outline color="primary" onClick={() => setCreateDatabaseMode("fromBackup")}>
                             Create one from backup
```

Comment:
Same here, imho - better to be consistent and have:
`Create one from backup => Restore one from backup`